yoda/executor/executor_test.go

```diff
@@ -8,36 +8,36 @@ import (
 )
 
 func TestParseExecutor(t *testing.T) {
-	name, url, timeout, err := parseExecutor("beeb:www.beebprotocol.com?timeout=3s")
-	require.Equal(t, name, "beeb")
+	name, url, timeout, err := parseExecutor("test:www.testprotocol.com?timeout=3s")
+	require.Equal(t, name, "test")
 	require.Equal(t, timeout, 3*time.Second)
-	require.Equal(t, url, "www.beebprotocol.com")
+	require.Equal(t, url, "www.testprotocol.com")
 	require.NoError(t, err)
 
-	name, url, timeout, err = parseExecutor("beeb2:www.beeb.com/anna/kondanna?timeout=300ms")
-	require.Equal(t, name, "beeb2")
+	name, url, timeout, err = parseExecutor("test2:www.test.com/anna/kondanna?timeout=300ms")
+	require.Equal(t, name, "test2")
 	require.Equal(t, timeout, 300*time.Millisecond)
-	require.Equal(t, url, "www.beeb.com/anna/kondanna")
+	require.Equal(t, url, "www.test.com/anna/kondanna")
 	require.NoError(t, err)
 
-	name, url, timeout, err = parseExecutor("beeb3:https://bandprotocol.com/gg/gg2/bandchain?timeout=1s300ms")
-	require.Equal(t, name, "beeb3")
+	name, url, timeout, err = parseExecutor("test3:https://bandprotocol.com/gg/gg2/bandchain?timeout=1s300ms")
+	require.Equal(t, name, "test3")
 	require.Equal(t, timeout, 1*time.Second+300*time.Millisecond)
 	require.Equal(t, url, "https://bandprotocol.com/gg/gg2/bandchain")
 	require.NoError(t, err)
 }
 
 func TestParseExecutorWithoutRawQuery(t *testing.T) {
-	_, _, _, err := parseExecutor("beeb:www.beebprotocol.com")
+	_, _, _, err := parseExecutor("test:www.testprotocol.com")
 	require.EqualError(t, err, "invalid timeout, executor requires query timeout")
 }
 
 func TestParseExecutorInvalidExecutorError(t *testing.T) {
-	_, _, _, err := parseExecutor("beeb")
-	require.EqualError(t, err, "invalid executor, cannot parse executor: beeb")
+	_, _, _, err := parseExecutor("test")
+	require.EqualError(t, err, "invalid executor, cannot parse executor: test")
 }
 
 func TestParseExecutorInvalidTimeoutError(t *testing.T) {
-	_, _, _, err := parseExecutor("beeb:www.beebprotocol.com?timeout=beeb")
-	require.EqualError(t, err, "invalid timeout, cannot parse duration with error: time: invalid duration \"beeb\"")
+	_, _, _, err := parseExecutor("test:www.testprotocol.com?timeout=test")
+	require.EqualError(t, err, "invalid timeout, cannot parse duration with error: time: invalid duration \"test\"")
```

Comment:
replace all testprotocol to bandprotocol instead.

Reply:
fixed